doc/terminology.rst

```diff
@@ -0,0 +1,51 @@
+.. _terminology:
+
+.. https://github.com/pydata/xarray/issues/2410
+.. https://github.com/pydata/xarray/issues/1295
+
+Terminology
```

Comment:
Glossary?

Reply:
Maybe? When I hear "glossary", I think of an alphabetized resource that I can ignore until I need it. This is a slightly more opinionated glossary designed to introduce terms in a specific order to new users. That said, I don't have a strong opinion and don't know if "terminology" is any better.

Reply:
:+1: